client/src/components/UploadImages.js

```diff
@@ -0,0 +1,39 @@
+import React, { useCallback } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+const UploadImages = (props) => {
+    const { setImages, images } = props;
+
+    const onDrop = useCallback(acceptedFiles => {
+        setImages(acceptedFiles)
+    }, [setImages])
+
+    const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop })
+    return (
+        <React.Fragment>
+            <h3>Upload images</h3>
```

Comment:
you can use material-ui's `<Typography variant="h3">Upload Images</Typography>` here

Reply:
Thanks!